src/externalized/ArmourModel.cc

```diff
@@ -1,8 +1,25 @@
 #include "ArmourModel.h"
 #include "Exceptions.h"
+#include "GamePolicy.h"
 #include "Item_Types.h"
 #include "Weapons.h"
 #include <cstdint>
+#include <set>
+
+static std::set<UINT16> const platesCanBeAttached = {
+	FLAK_JACKET,
+	FLAK_JACKET_18,
+	FLAK_JACKET_Y,
+	KEVLAR_VEST,
+	KEVLAR_VEST_18,
+	KEVLAR_VEST_Y,
+	KEVLAR2_VEST,
+	KEVLAR2_VEST_18,
+	KEVLAR2_VEST_Y,
+	SPECTRA_VEST,
+	SPECTRA_VEST_18,
+	SPECTRA_VEST_Y
+};
```

Comment:
This is another good candidate for externalization.

Reply:
Yes, this should become part of the JSON in the future.